### PR TITLE
Clean solution and read me

### PR DESCRIPTION
**Pull request by AmauryMi8**
All nuget packages are up to date.
The Api (puzzle) was removed from the solution.
The ReadMe was updated with new informations, like the Api documentation link. 
Tested on android device.

### CODE REVIEW

File: Challenge.Android/Properties/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="be.mi8.challenge.challenge">
-	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="29" />
+	<uses-sdk android:minSdkVersion="25" android:targetSdkVersion="29" />
```
Comment (Jeykairis) [inline Challenge.Android/Properties/AndroidManifest.xml:3]: Why set min sdk version to 25 ?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# 🏆 Complete the challenge and join the team !
+# 🏆 Complete the challenge and join the team !!
```
Comment (Jeykairis) [inline README.md:1]: Is a second ! a typo ?

**Review by Jeykairis — APPROVED** [review]
I tested the api https://numberfinderapi.azurewebsites.net and the chosen number is shared for multiple users. Can you fix it ?

**Comment by Jeykairis** [comment]
Also, can you test it on iOS ?

File: Challenge.Android/Properties/AndroidManifest.xml (hunk 1 above)
Comment (AmauryMi8) [reply Challenge.Android/Properties/AndroidManifest.xml:3]: I had error when deploying on device. Can not use Fast deployment option on an android device less than version 25. 
Fast deployment was not activated and my device android version was 29 so I bump this value up to 25 and error was gone.

File: README.md (hunk 2 above)
Comment (AmauryMi8) [reply README.md:1]: Yes, it is a typo, i remove it.

**Pull request merged**